docs/admin/compatibility_matrix.md

```diff
@@ -0,0 +1,10 @@
+# Compatibility Matrix
+
+While that last supported version will not be strictly enforced--via the max_version setting, any issues with an updated Nautobot supported version in a minor release, will require a bug to be raised and a fix in Nautobot core to address, with no fixes expected in this plugin. This allows the Nautobot Single Source of Truth App the ability to quickly take advantage of the latest features.
```

Comment:
I think this is boilerplate text, but I honestly have no idea what this paragraph is telling me.

Reply:
This was already in the repo before I created this PR.